utils/driverless.c

```diff
@@ -744,6 +795,9 @@ int main(int argc, char*argv[]) {
       }else if (!strcasecmp(argv[i], "_ipp._tcp")) {
 	/* reg_type_no = 0 for IPP entries only*/
 	reg_type_no = 0;
+      }else if (!strcasecmp(argv[i], "dns-sd-off")) {
```

Comment:
Could you call this option `--std-ipp-uris`. This conforms more to the standard of arguments which are options.